DTIPrep/DTIPrepRegister.pl

```diff
@@ -504,14 +504,13 @@ =head3 register_minc($minc, $raw_file, $data_dir, $inputs, ...)
   - $raw_file              : source file of the MINC file to register
   - $data_dir              : data_dir directory from the config table
   - $inputs                : input files of the file to be registered
-  - $pipelineName          : name of the pipeline used to obtain the
-                              MINC file
-  - $toolName              : tool name & version used
+  - $pipelineName          : name of the pipeline used to obtain the MINC file
+  - $toolName              : tool name and version used
   - $registeredXMLFile     : registered DTIPrep XML report
-  - $registeredQCReportFile: registered DTIPrep txt report
+  - $registeredQCReportFile: registered DTIPrep text report
   - $scanType              : scan type of the MINC file to register
-  - $registered_nrrd       : optional, registered NRRD file used to
-                              create the MINC file
+  - $registered_nrrd       : optional, registered NRRD file used to create
```

Comment:
you seem to be uniformly changing these to be one line per one input, so maybe you want to do the same here?

Reply:
done